src/SourceBuild/content/test/Microsoft.DotNet.SourceBuild.SmokeTests/assets/baselines/ArtifactsSizes/centos.8-x64.txt

```diff
@@ -0,0 +1,3849 @@
+./packs/Microsoft.AspNetCore.App.Ref/VERSION/analyzers/dotnet/cs/ru/Microsoft.Extensions.Configuration.Binder.SourceGeneration.resources.dll: 8192 bytes
+./packs/Microsoft.AspNetCore.App.Ref/VERSION/analyzers/dotnet/cs/tr/Microsoft.Extensions.Configuration.Binder.SourceGeneration.resources.dll: 7168 bytes
+./packs/Microsoft.AspNetCore.App.Ref/VERSION/analyzers/dotnet/cs/zh-Hans/Microsoft.Extensions.Configuration.Binder.SourceGeneration.resources.dll: 7168 bytes
+./packs/Microsoft.AspNetCore.App.Ref/VERSION/analyzers/dotnet/cs/zh-Hant/Microsoft.Extensions.Configuration.Binder.SourceGeneration.resources.dll: 7168 bytes
+./packs/Microsoft.AspNetCore.App.Ref/VERSION/analyzers/dotnet/roslyn4.4/cs/cs/Microsoft.Extensions.Logging.Generators.resources.dll: 10240 bytes
+./packs/Microsoft.AspNetCore.App.Ref/VERSION/analyzers/dotnet/roslyn4.4/cs/de/Microsoft.Extensions.Logging.Generators.resources.dll: 10752 bytes
+./packs/Microsoft.AspNetCore.App.Ref/VERSION/analyzers/dotnet/roslyn4.4/cs/es/Microsoft.Extensions.Logging.Generators.resources.dll: 10240 bytes
+./packs/Microsoft.AspNetCore.App.Ref/VERSION/analyzers/dotnet/roslyn4.4/cs/fr/Microsoft.Extensions.Logging.Generators.resources.dll: 10752 bytes
+./packs/Microsoft.AspNetCore.App.Ref/VERSION/analyzers/dotnet/roslyn4.4/cs/it/Microsoft.Extensions.Logging.Generators.resources.dll: 10240 bytes
+./packs/Microsoft.AspNetCore.App.Ref/VERSION/analyzers/dotnet/roslyn4.4/cs/ja/Microsoft.Extensions.Logging.Generators.resources.dll: 10752 bytes
+./packs/Microsoft.AspNetCore.App.Ref/VERSION/analyzers/dotnet/roslyn4.4/cs/ko/Microsoft.Extensions.Logging.Generators.resources.dll: 10240 bytes
+./packs/Microsoft.AspNetCore.App.Ref/VERSION/analyzers/dotnet/roslyn4.4/cs/Microsoft.Extensions.Logging.Generators.dll: 64000 bytes
+./packs/Microsoft.AspNetCore.App.Ref/VERSION/analyzers/dotnet/roslyn4.4/cs/pl/Microsoft.Extensions.Logging.Generators.resources.dll: 10240 bytes
+./packs/Microsoft.AspNetCore.App.Ref/VERSION/analyzers/dotnet/roslyn4.4/cs/pt-BR/Microsoft.Extensions.Logging.Generators.resources.dll: 10240 bytes
+./packs/Microsoft.AspNetCore.App.Ref/VERSION/analyzers/dotnet/roslyn4.4/cs/ru/Microsoft.Extensions.Logging.Generators.resources.dll: 12288 bytes
+./packs/Microsoft.AspNetCore.App.Ref/VERSION/analyzers/dotnet/roslyn4.4/cs/tr/Microsoft.Extensions.Logging.Generators.resources.dll: 10240 bytes
+./packs/Microsoft.AspNetCore.App.Ref/VERSION/analyzers/dotnet/roslyn4.4/cs/zh-Hans/Microsoft.Extensions.Logging.Generators.resources.dll: 9728 bytes
+./packs/Microsoft.AspNetCore.App.Ref/VERSION/analyzers/dotnet/roslyn4.4/cs/zh-Hant/Microsoft.Extensions.Logging.Generators.resources.dll: 9728 bytes
+./packs/Microsoft.AspNetCore.App.Ref/VERSION/data/FrameworkList.xml: 33002 bytes
+./packs/Microsoft.AspNetCore.App.Ref/VERSION/data/PackageOverrides.txt: 6616 bytes
+./packs/Microsoft.AspNetCore.App.Ref/VERSION/ref/net8.0/Microsoft.AspNetCore.Antiforgery.dll: 23552 bytes
```

Comment:
net8.0 should be abstracted away so that the baseline file is portable between releases.  Look at using the RemoveNetTfmPaths method in the BaselineHelper.

Reply:
Parsed out version and target rid information. Here is an [example ](https://github.com/dotnet/installer/pull/17233/files#diff-d1917e3c15e7ec81548e4a42961081dfb091f0d9ba291910ec9fffdabc95c2e6R4)of the output in the updated baseline.